frontend/packages/yki/src/interfaces/publicRegistration.ts

```diff
@@ -0,0 +1,16 @@
+export interface PublicRegistrationFillOutDetails {
+  address: string;
+  postNumber: string;
+  postOffice: string;
+  email: string;
+  emailConfirmation: string;
+  phoneNumber: string;
+  privacyStatementConfirmation: boolean;
+  certificateLanguage: string;
+  termsAndConditionsAgreed: boolean;
+}
+
+export interface PublicRegistration extends PublicRegistrationFillOutDetails {
+  firstNames: string;
+  lastName: string;
+}
```

Comment:
Viittaako tää jaottelu nyt siihen että suomi.fi-tunnistautumisen kautta tulisi mm. nuo nimitiedot suoraan, ja muut tiedot olisi käyttäjän täytettävissä? Jos näin, niin kannattaa valmistautua myös siihen sähköpostitunnistautumiseen, jolloin ainoa varmennettu tieto on käyttäjän sähköpostiosoite.

Reply:
Muita kenttiä mitä ilmolomakkeella löytyy:
- kansalaisuus
- syntymäaika
- hetu (valinnainen)
- sukupuoli (valinnainen)

Reply:
Joo, juuri noin et osa tulee valmiina ja osa sitten käyttäjän täytettävissä.